src/core/qgsmaprenderercache.cpp

```diff
@@ -105,7 +105,7 @@ bool QgsMapRendererCache::init( const QgsRectangle &extent, double scale )
   return false;
 }
 
-bool QgsMapRendererCache::updateParameters( const QgsRectangle &extent, const QgsMapToPixel &mtp )
+bool QgsMapRendererCache::updateParameters( const QgsRectangle &extent, const QgsMapToPixel &mtp, const QgsDateTimeRange &timeRange )
```

Comment:
you need to reset mTimeRange in  QgsMapRendererCache::init too (in deprecated function)

or @nyalldawson do you think we can remove the deprecated function at all ? I am starting to have doubts if it works at all now and QgsMapRendererCache is not something you use from plugins..